src/SprykerEco/Yves/Braintree/Dependency/Client/BraintreeToShipmentClientBridge.php

```diff
@@ -28,10 +29,10 @@ public function __construct($shipmentClient)
     /**
      * @param \Generated\Shared\Transfer\QuoteTransfer $quoteTransfer
      *
-     * @return \Generated\Shared\Transfer\ShipmentMethodsTransfer
+     * @return \Generated\Shared\Transfer\ShipmentMethodsCollectionTransfer
      */
-    public function getAvailableMethods(QuoteTransfer $quoteTransfer): ShipmentMethodsTransfer
+    public function getAvailableMethods(QuoteTransfer $quoteTransfer): ShipmentMethodsCollectionTransfer
     {
-        return $this->shipmentClient->getAvailableMethods($quoteTransfer);
+        return $this->shipmentClient->getAvailableMethodsByShipment($quoteTransfer);
```

Comment:
This is also a BC break as this method doesn't exist in the old versions.

Reply:
yes. the task was about fixing dependency with latest shipment